core/docs/API_Python.md

```diff
@@ -0,0 +1,160 @@
+SPIRIT API
+====================
+
+Python API
```

Comment:
This is not needed. Instead, promote the other ### Headings to this level (as in the other file).